cluster/plan.go

```diff
@@ -598,6 +835,119 @@ func (c *Cluster) BuildKubeletProcess(host *hosts.Host, prefixPath string, svcOp
 	}
 }
 
+func (c *Cluster) BuildClusterKubeProxyProcess(osType string, svcOptionData map[string]*v3.KubernetesServicesOptions) v3.RKEProcess {
+	isWindows := osType == "windows"
+
+	envMap := map[string]sets.Empty{}
+	bindsMap := map[string]sets.Empty{}
+
+	for _, env := range c.Services.Kubeproxy.ExtraEnv {
```

Comment:
maybe we can extract this logic of populating envMap and bindsMap for all processes?